examples/seldon_deployment/README.md

```diff
@@ -398,45 +387,40 @@ configure it, you will need to set up credentials explicitly in the ZenML secret
 e.g.:
 
 ```bash
-$ zenml secret register -s seldon_s3 s3-store
-You have supplied a secret_set_schema with predefined keys. You can fill these
-out sequentially now. Just press ENTER to skip optional secrets that you do not
-want to set
-Secret value for rclone_config_s3_type:
-Secret value for rclone_config_s3_provider:
-Secret value for rclone_config_s3_env_auth: False
-Secret value for rclone_config_s3_access_key_id: ASAK2NSJVO4HDQC7Z25F
-Secret value for rclone_config_s3_secret_access_key: AhkFSfhjj23fSDFfjklsdfj34hkls32SDfscsaf+
-Secret value for rclone_config_s3_session_token: AFdfsaSf2SDFfdaWAsfCacs...ASDFsfdfs23sc==
-Secret value for rclone_config_s3_region: us-east-1
-Secret value for rclone_config_s3_endpoint:
+$ zenml secret register -s seldon_s3 s3-store \
+    --rclone_config_s3_env_auth=False \
+    --rclone_config_s3_access_key_id='ASAK2NSJVO4HDQC7Z25F' \ --rclone_config_s3_secret_access_key='AhkFSfhjj23fSDFfjklsdfj34hkls32SDfscsaf+' \
+    --rclone_config_s3_session_token=@./aws_session_token.txt \
+    --rclone_config_s3_region=us-east-1
```

Comment:
Should these really be explicitly set inside the README?

Reply:
These are dummy values, not actual key values, if that's what you are concerned about.